.dockerignore

```diff
@@ -0,0 +1 @@
+Dockerfile
```

Comment:
I'm not sure we want the top-level `Dockerfile` to be the bindings generator; I can imagine other Docker workflows that we might end up wanting in the repo. Could this be moved to a sub-directory?

Reply:
Moved to `tools/` updated `tools/README.md` and the command in `docs/updating-sdk.md`.